### DISCUSSION TITLE
Contributor call: 8 January 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#1616] [Support] [answered]
You are all invited to the contributor call. 
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

Welcome to 2022 😊

Organized by @tangerineshark and @lucibytes.

### About
- Location: https://discord.freesewing.org (meeting room: `🔊voice chat`)
- Date: Saturday January 8, 2022 16:00 UTC, which is:
        Los Angeles: 09:00
        Chicago: 11:00
        New York: 12:00
        São Paulo: 14:00
        London: 17:00
        Paris: 18:00
        Johannesburg: 19:00
        Moscow: 20:00
        Mumbai: 22:30
        Hong Kong: 01:00 on Sunday
        Tokyo: 02:00 on Sunday
        Sydney: 04:00 on Sunday
Or check the bot in Discord! `#meetings`

### Agenda
- Newsletter
- Status of patterns in development
- Poncho pattern

### Notes
- Newsletter update by @joostdecock, most notably: frontend code migrated from GatsbyJS to NextJS, which helps for the newly deployed dev site, it is now easier to re-use code between that site and the regular FreeSewing site, which makes maintenance easier.
- Waralee documentation is on the way.
- @joostdecock pruned a lot of Github issues: those that were older than a year, those that were just “this would be a nice feature”, and a lot of issues were transferred to discussions or ideas. It is now a manageable to-do list of approximately 25 issues.
- @bobgeorgethe3rd and @joostdecock: it would be nice if for every pattern there would be an ambassador who would make sure to keep their pattern up-to-date, including documentation. We already have a system in place for when a pattern falls apart front-end: an issue will be made and the developer of the pattern will automatically be pinged (or it defaults to pinging @joostdecock) - @bobgeorgethe3rd will be added for Holmes.
- @tangerineshark asked how to best co-develop a newsletter.
    - @joostdecock: a deadline is very important, if it’s missed once, it will likely be slacked on in the future. If things aren’t finished before it, then @joostdecock will fill in the gaps, or send out a shorter newsletter.
    - If people have ideas, they can write it and put it in Strapi, the rest will be handled. The newsletter is done manually, since it’s only needed every three months.
    - On the question of outsourcing content: people are more motivated to write articles on things they’re passionate about, not “we need this and this for the newsletter”.
- @darigovresearch with the idea of making a poncho pattern by recycling one of the hoods. Question is, which pattern are we taking the hood from?
    - FreeSewing basically has two types of hoods: a 2-piece (Huey’s) and a 3-piece (Hugo’s and Yuri’s).
    - @joostdecock: the raglan sleeves make it complicated to take the hood from Hugo’s pattern. Huey is probably the best option.
    - Perhaps it’s possible to add the option to switch between different types of hoods.
- @darigovresearch with the question of what exactly is needed to submit a pattern to FreeSewing (in regards to the bib pattern).
    - @nataliasayang and @starfetch emphasized not to worry too much about the last step, to just follow the tutorial and focus on the steps of making the pattern, and that everything will be figured out later.
    - @joostdecock further explained the two options for submitting a pattern, one is the ‘quickstart’ FreeSewing way (a standalone repository), the other is to fork the monorepo and put it all in there, then do a pull request. But if you’re just starting out, don’t worry about that. @starfetch added that the easier thing about the monorepo way is that there aren’t dependencies to worry about.
    - The conclusion: publish a standalone repository for the pattern, @joostdecock will import it.
- @bobgeorgethe3rd’s updates on the Laura leggings
    - Seat measurement is now completely in there and he has remade the pattern’s top part entirely.
    - Added an option for front curve depth, to make the curve tighter or less tight.
    - There are still problems when the seat and upper leg measurements get ‘close’ to each other.
    - @tangerineshark mentioned that the (default) curves of the crotch seem too shallow.
    - @joostdecock suggested to base it off of Titan as making a pants pattern from scratch is a *lot* of work and that a deeper curve is indeed needed. @bobgeorgethe3rd may switch over.
    - @bobgeorgethe3rd: what was a ‘simple leggings pattern’ devolved into a lot more. There was a version with a deeper curve, but there was another problem there. The problem with basing it off of Titan is that for this pattern, you don’t want the outer seam. @joostdecock suggested to start from Titan without the outer seam, this will still be easier.
    - @bobgeorgethe3rd will probably take a break from this pattern for a while.

### To-Do's
- Add @bobgeorgethe3rd for the Holmes pattern
- Figure out which hood for the poncho pattern

**Comment by darigovresearch** [comment]
@lucibytes & @tangerineshark thanks for organising this next call!

Do you use some automated tool to generate the time zones?

Would be helpful to know for other open source projects

**Comment by l3xander** [comment, marked as answer]
Hi @darigovresearch, thanks!

I copied the list from @joostdecock and I just maintain it manually (which has its flaws). Maybe he has a better answer.

But for Discord we use [the sesh bot](https://sesh.fyi/) to convert timezones, or well, Discord has a built-in feature for it too nowadays ('events'). Helpful for projects with a Discord server 😋

**Comment by darigovresearch** [comment]
@lucibytes you're very welcome & thanks for the explanation. 

Definitely seems like it's something automatable to help make it easier for those running the contributor calls and/or some other tool already exists for what you need

**Comment by darigovresearch** [comment]
Can we add to the agenda a discussion for the poncho pattern ?  - https://github.com/freesewing/freesewing/discussions/1618

Namely which specific pattern do contributors recommend forking from to make the poncho hood [huey](https://freesewing.org/designs/huey/), [hugo](https://freesewing.org/designs/hugo/) or [Yuri](https://freesewing.org/designs/yuri/)

**Reply by l3xander** [reply]
Done! 😄👍

**Comment by darigovresearch** [comment]
@lucibytes thanks for the great notes that were added to this contributor call. 

We'd like to help with the admin side of the contributor calls. @lucibytes & @tangerineshark would you be up for letting us set up an initial discussion for the 22nd where you can correct and adjust/correct as required ?

**Comment by l3xander** [comment]
Hey @darigovresearch, thank you! 😄
If you insist :)

**Comment by joostdecock** [comment]
I wanted to send the link to the next contributor call to someone, but there was nothing to point them to.
So I created it: https://github.com/freesewing/freesewing/discussions/1687

:shrug:

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
